client-v2/src/shared/actions/__tests__/editionscards.spec.ts

```diff
@@ -0,0 +1,51 @@
+import configureMockStore from 'redux-mock-store';
```

Comment:
Thanks for the great work on the tests here, the scaffolding is non-trivial and the specs look good 👍

Reply:
Most of it is refactored pre-existing tests  - I split them out for clarity so that I could add *one* cleanly.

Reply:
Ah, gotcha 👍